src/cascadia/ut_app/DynamicProfileTests.cpp

```diff
@@ -1,673 +0,0 @@
-// Copyright (c) Microsoft Corporation.

```

Comment:
these guys didn't really get resurrected anywhere, did they?

Reply:
`TestGenGuidsForProfiles` was moved into `ProfileTests.cpp`.
I've removed the others because the `SettingsLoader` makes no distinction between the source of builtin profiles: "inbox" JSON data and generated profiles are treated the exact same way.
Considering the extensive coverage of inbox JSON related tests in `SerializationTests.cpp` I've thus decided to not port these tests to the new API and saved myself some work.

Would you like me to restore the tests?